lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -554,6 +601,8 @@ def main():
             result = create_stack(module, stack_params, cfn)
         elif module.params.get('create_changeset'):
             result = create_changeset(module, stack_params, cfn)
+        elif module.params.get('termination_protection') is not None:
```

Comment:
This is working almost as expected for me. It seems like now general stack updates will not occur unless termination_protection is None.

I'm wondering if update_stack() should call update_termination_protection().

That would also allow the exit output to stay the same if no update is being made - with this change if I call the module twice with termination_protection: false then I get the message:
`"msg": "Termination protection on stack ... is already the requested value"` whereas normally if a stack has no updates we would get the output: `"output": "Stack is already up-to-date."`.

Reply:
Oh, good catch and thanks for the review. This will actually break one of my own playbooks as I create a stack and register the output for use in subsequent tasks. I will make the required changes and let you know. 👍